src/Post.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Comment from './Comment';
+import {connect} from 'react-redux';
+import * as post from './store/post';
+
+
+const Post = ({ ownpost, post, setPost }) => {
+  const removePost = () => {
+    setPost(post.filter(post => post.id !== ownpost.id));
+  }
+
+  return (
+    <>
+      <button type="button" className="remove_post" onClick={removePost}>
```

Comment:
тут має бути щось таке
```suggestion
      <button type="button" className="remove_post" onClick={() => removePost(post.id)}>
```